Beat-Board/requirements.txt

```diff
@@ -0,0 +1,5 @@
+The following modules are required to run beatBoard.py
```

Comment:
wrong way of defining requirements.txt file. See this example: https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Amazon-Price-Alert/requirements.txt